index.html

```diff
@@ -440,6 +463,7 @@ <h3 id="cdn-paas"><a href="#cdn-paas">CDN &amp; PaaS performance</a></h3>
             <td class="ok">yes</td>
             <td class="ok">yes</td>
             <td class="ok"><a href="https://blog.cloudflare.com/introducing-http2/">yes</a></td>
+            <td class="ok"><a href="https://blog.cloudflare.com/http3-the-past-present-and-future/">no</a></td>
```

Comment:
```suggestion
            <td class="ok"><a href="https://blog.cloudflare.com/http3-the-past-present-and-future/">yes</a></td>
```
:)

Reply:
I put no because we don't support the yet-to-be-published RFC version of HTTP/3. However, the class should be `alert` not `ok`, which I just fixed up.